app/client/src/features/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,224 @@
+        }
+    });
+
+    return (
+        // <FadeThrough animKey='dashboard-page'>
+        <Grid container>
+            <Grid item xs={12} className={classes.item}>
+                <Card className={classes.card}>
+                    <CardContent>
+                        <Typography variant='h6' className={classes.title}>
+                            Current Events
+                        </Typography>
+                        {currentEvents.length > 0 ? (
+                            <Grid container item direction='column'>
+                                <Grid item xs={12}>
+                                    <List>
+                                        {currentEvents.map(({ node: event }, idx) => {
+                                            if (event?.id && event?.description && event?.title && event?.startDateTime && event?.organization?.name) {
+                                                return (
+                                                    <ListItem
+                                                        button
+                                                        key={event?.id}
+                                                        divider={idx !== currentEvents.length - 1}
+                                                        onClick={handleNav(`/events/${event?.id}/settings`)}
+                                                    >
+                                                        <DashboardEvent
+                                                            key={event.id}
+                                                            id={event.id}
+                                                            title={event.title}
+                                                            description={event.description}
+                                                            startDateTime={event.startDateTime}
+                                                            organization={event.organization.name}
+                                                        />
+                                                        <Link onClick={handleNav(`/events/${event?.id}/live`)}>
+                                                            <Button
+                                                                aria-label='view live feed of current event'
+                                                                variant='contained'
+                                                                color='primary'
+                                                            >
+                                                                Live Feed
+                                                            </Button>
+                                                        </Link>
+                                                    </ListItem>
+                                                );
+                                            }
+                                        })}
```

Comment:
Considering this logic is duplicated, I think it may be worthwhile to put it into its own component. Maybe `DashboardEventListItem` where you could just do 
```tsx
<List>
  {currentEvents.map((event) =>  <DashboardEventListItem event={event} />}
</List>
```

Also, add any other configurable prop you might need.